source/Nevermore/Advanced/RelationalTransactionRegistry.cs

```diff
@@ -69,16 +69,13 @@ string BuildHighNumberOfTransactionsMessage()
 
         public void WriteCurrentTransactions(StringBuilder sb)
         {
-            ReadTransaction[] copy;
-            lock (transactions)
```

Comment:
Locking here is not necessary, the ConcurrentDictionary will grab a lock when reading the keys.